libs/libc/machine/arm/gnu/arch_setjmp.S

```diff
@@ -64,7 +59,10 @@
  *
  ****************************************************************************/
 
+#ifdef CONFIG_ARM_THUMB
+	.thumb
```

Comment:
Do we really need `.thumb` here?

Reply:
It can be removed, it just a explicit description (a alias to `.code 16`) for thumb mode.

Reply:
so, let's remove it?

Reply:
OK, removed